torch/csrc/distributed/c10d/quantization.h

```diff
@@ -0,0 +1,21 @@
+// (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+
+#pragma once
+
+
+#include <ATen/ATen.h>
+
+void FloatToBFloat16Quantized_ref(
+    const float* const input,
+    const size_t nrows,
+    const size_t ncols,
+    uint16_t* const output);
+
+void BFloat16QuantizedToFloat_ref(
+    const at::BFloat16* const input,
+    const size_t nrows,
+    const size_t ncols,
+    float* const output);
+
+at::Tensor _float_to_bfloat16_cpu(const at::Tensor& input);
```

Comment:
let's wrap the exposed functions in some namespace, probably not `at::` as we don't want to conflict with internals, maybe something like `torch::distributed::c10d::quantization`?